src/components/dashboard/components/dictionary/DictionaryContainer.jsx

```diff
@@ -204,42 +207,40 @@ export class DictionaryOverview extends Component {
           <div className="text-center mt-3" id="loader">
             <Loader />
           </div>
-        )
-          : (
-            <div className="dashboard-wrapper">
-              <DictionaryDetailCard
-                dictionary={this.props.dictionary}
-                versions={this.props.versions}
-                cielConcepts={cielConcepts}
-                customConcepts={customConcepts}
-                diagnosisConcepts={diagnosisConcepts}
```

Comment:
Is this a formatting change?

Reply:
After I pulled from the main master and rebased. I had to fix some merge conflicts that is why I touched this file.